stylesheets/bootstrap.scss

```diff
@@ -72,4 +72,13 @@
 // Custom breadcrumb separator
 .breadcrumb > li + li::before {
     content: ">"
-}
\ No newline at end of file
+}
+
+pre {
+    overflow-x: auto;
+}
+
+pre code {
+    overflow-wrap: normal;
```

Comment:
I can appreciate that 😁 I think that's the most common result when my code is being reviewed -- I was too lazy.